include/rom_interfaceform.hpp

```diff
@@ -10,12 +10,16 @@
 #include "hdf5_utils.hpp"
 #include "hyperreduction_integ.hpp"
 #include "linalg/NNLS.h"
+#include "etc.hpp"
 
 namespace mfem
 {
 
 class ROMInterfaceForm : public InterfaceForm
 {
+private:
+   mutable TimeProfiler timer;
```

Comment:
This is already defined in the parent class.

Reply:
I intentionally declared them as private, so each class would use separate timers. Otherwise at the destruction of `ROMInterfaceForm` the time profile is transferred to the parent `timer` and will be printed there as well, which make us confusing.
I wish I'd be able to not print timer of `~InterfaceForm` if it is called from `~ROMInterfaceForm`. Do you happen to know how I should implement this?

Reply:
I see, that is confusing. Having `timer` in the parent and derived classes can also be a little confusing. One option is a virtual function for writing output for timers, instead of relying on the destructor. Maybe the virtual function for timer output could be called by the destructor. Another option might be `dynamic_cast` to see whether it is an instance of the derived class.

Reply:
`TimeProfiler` now has its private member `std::string title`. At its destruction it will print the time table with its own title. At instantiation we can set this title, usually the name of the class that uses this timer. Now `ROMInterfaceForm` inherits `timer` from `InterfaceForm` and changes its title to `ROMInterfaceForm`. So there will be only one timer but only one time table will be printed with the right class name, without confusion.